sql/src/main/java/org/apache/druid/sql/calcite/rel/Windowing.java

```diff
@@ -270,6 +278,74 @@ public static Windowing fromCalciteStuff(
     }
   }
 
+  /**
+   * A wrapper object which stores {@link org.apache.calcite.rel.core.Window.Group}
+   * along with its computed {@link WindowOperatorFactory}
+   *
+   * this allows us to sort the window groups in order to optimise the order of operators we would need to compute
+   * without losing the aggregate column name information (which is part of the computed WindowOperatorFactory)
+   */
+  private static class WindowGroupProcessorWrapper implements Comparable<WindowGroupProcessorWrapper>
```

Comment:
I would really like to suggest a better name for it - but don't really have any good suggestions.... `WindowedCalculation` ?

Reply:
I changed it to `WindowComputationProcessor`. let me know your thoughts on this name